app/templates/search/macros/filters/collections_tna.html

```diff
@@ -0,0 +1,38 @@
+{% from 'components/button/macro.html' import tnaButton %}
+{% from 'components/checkboxes/macro.html' import tnaCheckboxes %}
+
+{% macro collections_tna(request, collection) %}
+<div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
+  {% set collections = [] %}
+  {% for collection_id, collection in collection.items() %}
+    {% set collections = collections.append({
+      'text': collection,
+      'value': collection_id,
+      'checked': qs_is_value_active(request.GET, 'collections', collection_id)
+    }) %}
+  {% endfor %}
+  {{ tnaCheckboxes({
+    'label': 'Collections',
+    'headingLevel': 3,
+    'headingSize': 'm',
+    'id': 'collections',
+    'name': 'collections',
+    'items': collections,
+    'small': True,
+    'attributes': {
+      'form': 'search-form'
+    }
+  }) }}
+  <div class="tna-button-group tna-!--margin-top-s">
+    {{ tnaButton({
+      'text': 'Update',
+      'buttonElement': True,
+      'buttonType': 'submit',
+      'small': True,
+      'attributes': {
+        'form': 'search-form'
+      }
+    }) }}
+  </div>
+</div>
```

Comment:
```suggestion
  <div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
    {% set collections = [] %}
    {% for collection_id, collection in collection.items() %}
      {% set collections = collections.append({
        'text': collection,
        'value': collection_id,
        'checked': qs_is_value_active(request.GET, 'collections', collection_id)
      }) %}
    {% endfor %}
    {{ tnaCheckboxes({
      'label': 'Collections',
      'headingLevel': 3,
      'headingSize': 'm',
      'id': 'collections',
      'name': 'collections',
      'items': collections,
      'small': True,
      'attributes': {
        'form': 'search-form'
      }
    }) }}
    <div class="tna-button-group tna-!--margin-top-s">
      {{ tnaButton({
        'text': 'Update',
        'buttonElement': True,
        'buttonType': 'submit',
        'small': True,
        'attributes': {
          'form': 'search-form'
        }
      }) }}
    </div>
  </div>
```
Missing indent. When indented it becomes easier to read and collapse the `{% macro` line.

Reply:
Hi @TNA-Allan 👋🏽 , thank you for the review on this and the requested change. 

I find these GitHub suggested change diffs quite difficult to read but - if I've understood correctly - you're suggestion is to routinely indent the HTML that appears within the macro body. I see what you mean about the indentation potentially making it easier to read (although PyCharm doesn't seem to provide me with an option to collapse even if it is indented). 

The thing is that, as far as I could see when working on this PR, we don't typically indent the root element of a filter macro's HTML, so my intention was here was to follow the indentation approach that has been established for other filter macros. Here are the macros I've based this observation on: 

1. [search_within.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/macros/filters/search_within.html) 
2. [levels_tna.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/macros/filters/levels_tna.html)
3. [date_range.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/macros/filters/date_range.html)
4. [closure_statuses.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/macros/filters/closure_statuses_tna.html)

That said, looking at other parts of the codebase in response to your suggestion, I can see that we do sometimes also indent. See:

1. [details_fields.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/records/macros/detail_fields.html)
2. [series_information.html](https://github.com/nationalarchives/ds-search/blob/main/app/templates/records/macros/series_information.html)

I don't have particularly strong feelings either way but I wonder if there _might_ be a case for consistency in macro HTML indentation across this codebase (or maybe not - perhaps it doesn't matter). 

What I'd like to do, if it's OK with you, is to leave the indentation as it is and get a view from @ahosgood on whether we'd want to aim for consistency in indentation. If so, I'll raise a PR that introduces consistent indentation across macros.